src/api/db/data/20220920115342_copy_old_watched_projects.rb

```diff
@@ -0,0 +1,7 @@
+class CopyOldWatchedProjects < ActiveRecord::Migration[6.1]
+  def up
+    WatchedProject.all.each do |old_watched_project|
+      WatchedItem.find_or_create_by(watchable: old_watched_project.project, user: old_watched_project.user)
```

Comment:
```suggestion
    WatchedProject.all.each do |old_watched_project|
      next if old_watched_project.user.watched_items.count.positive?
      
      WatchedItem.find_or_create_by(watchable: old_watched_project.project, user: old_watched_project.user)
```

Reply:
I wouldn't mess with peoples *new* watchlist

Reply:
I see. Suggestion applied. I also added the following sentence to the commit description: _"Import the projects only if the user doesn't watch any item in the new implementation."_ .

Please take another look.

Reply:
@hennevogel @eduardoj  :thinking: I'm not convinced of this decision...

We know some users made the effort to add all the projects they had in the old watchlist to the new one.

But some other users could have tried out the new watchlist by adding random projects/packages/requests and then they disabled the feature flag to continue using the possibly long list of projects they had in the old watchlist. 
In that case, I wouldn't like them to lose the list they had.

Reply:
> But some other users could have tried out the new watchlist by adding random projects/packages/requests and then they disabled the feature flag to continue using the possibly long list of projects they had in the old watchlist.
In that case, I wouldn't like them to lose the list they had.

Then you need to make this more complicated and add a feature to do the thing in between...